lib/Swagger/src/Discussion/Api/VotesApi.php

```diff
@@ -205,14 +194,15 @@ public function createUpVoteWithHttpInfo($site_id, $post_id)
                 $queryParams,
                 $httpBody,
                 $headerParams,
-                '\Swagger\Client\Discussion\Models\UpvoteHalResponse'
+                '\Swagger\Client\Discussion\Models\VotingResponse',
+                '/{siteId}/votes/post/{postId}'
             );
 
-            return array($this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\Discussion\Models\UpvoteHalResponse', $httpHeader), $statusCode, $httpHeader);
+            return array($this->apiClient->getSerializer()->deserialize($response, '\Swagger\Client\Discussion\Models\VotingResponse', $httpHeader), $statusCode, $httpHeader);
         } catch (ApiException $e) {
             switch ($e->getCode()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a "case default" clause to this "switch" statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS131)